### PR TITLE
Separate elements

### PR DESCRIPTION
**Pull request by PsychicMuffin**
Split up elements in UI and calculations, added select for shifts to pick element, updated calculations to be a bit more straightforward, updated encoding to support elements in tables. Also switched project code style to be in use to make sure styles match.

### CODE REVIEW

File: js/DataService.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    lightning: 2,
+    chaos: 3
+  };
+
+  var indexToElement = {
+    0: 'fire',
+    1: 'cold',
+    2: 'lightning',
+    3: 'chaos'
+  };
```
Comment (PsychicMuffin) [inline js/DataService.js]: I really wanted to find a better way to do this two way mapping stuff, but most solutions I ran across just didn't feel like they added enough to justify the complexity.

Comment (ficocelliguy) [reply js/DataService.js]: I think you could just make an array of the strings, then use indexOf('fire') to get the index, and square brackets to pull the string out using the index.

File: selectAllOnClick.js [hunk 2]
```diff
@@ -1,27 +0,0 @@
-angular.module('PathOfDamage')
```
Comment (ficocelliguy) [inline selectAllOnClick.js:1]: Did you mean to remove this?

**Review by ficocelliguy — COMMENTED** [review]
I'll be able to test this tomorrow.

File: selectAllOnClick.js (hunk 2 above)
Comment (PsychicMuffin) [reply selectAllOnClick.js:1]: yeah, you added it to js/ folder, but didn't remove it from the project root directory

**Review by ficocelliguy — APPROVED** [review]

**Pull request merged**